Modules/ITS/src/ITSClusterTask.cxx

```diff
@@ -180,6 +182,15 @@ void ITSClusterTask::monitorData(o2::framework::ProcessingContext& ctx)
 #endif
   // Filling cluster histogram for each ROF by open_mp
 
+  if (mDoPublishDetailedSummary == 1) {
```

Comment:
@lbariogl 

When I am building your code, I have the error: 

>error: for statement expected before 'if'
This error comes from the fact that OpenMP would require a "for loop" after the OpenMP pragma expression. Do you have a such error? Interestingly, it appeared only for my case. 

Could you move this new if statement before the OpenMP parallelization? For my case, it solved the issue

Reply:
@IsakovAD I have not tried, but you are right. I have moved it